code/components/jomjol_wlan/read_wlanini.cpp

```diff
@@ -157,8 +180,8 @@ int LoadWlanFromFile(std::string fn)
             #endif
         }
 
-        /* read next line */
-        if (fgets(zw, sizeof(zw), pFile) == NULL) {
+        if (fgets(zw, 1024, pFile) == NULL)
```

Comment:
why not sizeof(zw) - sounds better for me....

Reply:
Especially with `char zw[256];` above

Reply:
I agree, the initial change does not make sense.